src/libs/modulegroups.c

```diff
@@ -907,6 +907,7 @@ static void _preset_retrieve_old_presets(dt_lib_module_t *self)
 
     gchar *tx = _preset_retrieve_old_layout(list, fav);
     dt_lib_presets_add(pname, self->plugin_name, self->version(), tx, strlen(tx), FALSE);
+    g_free(tx);
```

Comment:
for some reason, freeing the string was crashing dt at some point. That's why I've not done that. But apparently this is no more the case :) maybe the problem was elsewhere (or something as changed in presets code)